src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug

```diff
@@ -1,6 +1,6 @@
 button.btn.btn-link.btn-icon.visible-xl.waves-notransition.active(ng-if="$root.inbox.rightSidebar.isVisible", ui-sref="^", title="{{ 'Hide emails with attachments' | translate }}")
   i.mdi.mdi-paperclip
-button.btn.btn-link.btn-icon.visible-xl.waves-notransition(ng-if="!$root.inbox.rightSidebar.isVisible", ui-sref=".attachments", title="{{ 'Show emails with attachments' | translate }}")
```

Comment:
That means that nowhere in the SPA we use the .attachments route anymore, so you should remove it from the router definition

Reply:
Yes, I will remove it